src/components/pages/Dashboard/Dashboard.js

```diff
@@ -87,7 +87,7 @@ const Dashboard = props => {
   if (tickets !== []) {
     escaTickets = tickets.filter(x => x.ticket_status === 'approved');
   }
-  console.log('escaTickets', escaTickets);
+  // console.log('escaTickets', escaTickets);
```

Comment:
Is there a reason for this commented-out code to remain?

Reply:
Great catch. This console.log was showing in the dev console. I believe it was accidentally left there, so I commented it out and pushed it. We'll have Paul take a look and see what he thinks.

Reply:
Sounds good.

Reply:
This console log precedes this ticket--so we hadn't tackled it--but we should probably delete it. I'm curious what Paul has to say.

Reply:
I actually tend to agree - we should all take responsibility for cleaning up our codebases when we see something like ugly commented-out code! Everyone pitching in is how huge companies operate and keep things running smoothly. 😉

Reply:
to me this seems like the commented out log is still in the code.